tests/test_parser.py

```diff
@@ -29,10 +29,14 @@ def wrap(str, element_form="qualified", attribute_form="unqualified"):
 
 
 class ParserTests(TestCase):
+    def setUp(self) -> None:
+        self.parser = SchemaParser()
+        super(ParserTests, self).setUp()
+
     def test_with_unknown_tags_raise_exception(self):
         xsd = "<xs:assert />"
         with self.assertRaises(NotImplementedError) as cm:
-            SchemaParser.from_string(wrap(xsd))
+            self.parser.from_string(wrap(xsd))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No value for argument 'clazz' in method call](https://app.codacy.com/manual/tefra/xsdata/pullRequest?prid=4930005)